From a2c8b03d2dc15792046fb474924c7366504b2201 Mon Sep 17 00:00:00 2001
From: Sandor Yu <R01008@freescale.com>
Date: Mon, 2 Feb 2015 13:54:23 +0800
Subject: [PATCH] MLK-10192: dts: Move sii902x hdmi function to imx6sx sdb
 lcdif1 dts

-Sii902x hdmi daughter connect to lcdif1 interface,
move this function to lcdif1 dts.
-Sii902x hdmi driver share the reset pin with ov5640 driver,
one driver will been reset by the other driver,
so move sii902x reset pin configure to licdif1 dts.

Signed-off-by: Sandor Yu <R01008@freescale.com>
---
 arch/arm/boot/dts/imx6sx-sdb-lcdif1.dts | 12 +++++++++++-
 arch/arm/boot/dts/imx6sx-sdb.dts        |  4 +++-
 2 files changed, 14 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sx-sdb-lcdif1.dts b/arch/arm/boot/dts/imx6sx-sdb-lcdif1.dts
index b066143..227f643 100644
--- a/arch/arm/boot/dts/imx6sx-sdb-lcdif1.dts
+++ b/arch/arm/boot/dts/imx6sx-sdb-lcdif1.dts
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2014 Freescale Semiconductor, Inc.
+ * Copyright (C) 2014-2015 Freescale Semiconductor, Inc.
  *
  * This program is free software; you can redistribute it and/or modify
  * it under the terms of the GNU General Public License version 2 as
@@ -18,6 +18,10 @@
 			status = "okay";
 		};
 	};
+
+	sii902x_reset: sii902x-reset {
+		status = "okay";
+	};
 };
 
 &csi1 {
@@ -27,3 +31,9 @@
 &lcdif1 {
 	status = "okay";
 };
+
+&i2c1 {
+	sii902x@39 {
+		status = "okay";
+	};
+};
diff --git a/arch/arm/boot/dts/imx6sx-sdb.dts b/arch/arm/boot/dts/imx6sx-sdb.dts
index 5204ac4..0a262ff4 100644
--- a/arch/arm/boot/dts/imx6sx-sdb.dts
+++ b/arch/arm/boot/dts/imx6sx-sdb.dts
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2014 Freescale Semiconductor, Inc.
+ * Copyright (C) 2014-2015 Freescale Semiconductor, Inc.
  *
  * This program is free software; you can redistribute it and/or modify
  * it under the terms of the GNU General Public License version 2 as
@@ -180,6 +180,7 @@
 		reset-gpios = <&gpio3 27 1>;
 		reset-delay-us = <100000>;
 		#reset-cells = <0>;
+		status = "disabled";
 	};
 };
 
@@ -607,6 +608,7 @@
 		bits-per-pixel = <32>;
 		resets = <&sii902x_reset>;
 		reg = <0x39>;
+		status = "disabled";
 	};
 };
 
-- 
1.8.0

